applications/ChatGPT/chatgpt/nn/reward_model.py

```diff
@@ -24,14 +24,14 @@ def __init__(self,
                  lora_train_bias: str = 'none') -> None:
         super().__init__(lora_rank=lora_rank, lora_train_bias=lora_train_bias)
         self.model = model
+        self.convert_to_lora()
```

Comment:
Why not convert value head?

Reply:
value head is random initialized, so we should not use lora to value head